framework/include/utils/RankTwoTensorImplementation.h

```diff
@@ -109,9 +110,26 @@ RankTwoTensorTempl<T>::RankTwoTensorTempl(const TypeVector<T> & row1,
   for (auto i : make_range(N))
     this->_coords[N + i] = row2(i);
 
-  const unsigned int two_n = N * 2;
   for (auto i : make_range(N))
-    this->_coords[two_n + i] = row3(i);
+    this->_coords[2 * N + i] = row3(i);
+}
+
+/// named constructor for initializing symmetrically
```

Comment:
Probably just delete this description since it's covered in the other .h file to avoid the two from getting out of sync.